extensions/amp-analytics/amp-analytics.md

```diff
@@ -137,6 +137,12 @@ Type attribute value: `cxense`
 
 Adds support for Cxense Insight analytics. Requires defining *var* `siteId` with Cxense-provided *siteId*. More details can be found at [wiki.cxense.com](https://wiki.cxense.com/display/cust/Accelerated+Mobile+Pages+%28AMP%29+integration).
 
+### Gemius
+
+Type attribute value: `gemius`
+
+Adds support for Gemius Audience/Prism analytics. Additionally, the gemius-provided `prefix` and `identifier` variables must be specified. It's also possible to specify the optional variable `extraparams` (key1=value1|key2=value2).
```

Comment:
are `|` and `=` supposed to encoded in extraUrlParams? If they are supposed to be encoded, please use `extraUrlParams` feature. if not, then this is fine.